okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpChannelBuilder.java

```diff
@@ -122,10 +123,23 @@ public OkHttpChannelBuilder overrideHostForAuthority(String host) {
   }
 
   /**
-   * Provides a SSLSocketFactory to establish a secure connection.
+   * Sets the negotiation type for the HTTP/2 connection.
+   *
+   * <p>Default: <code>TLS</code>
+   */
+  public OkHttpChannelBuilder negotiationType(NegotiationType type) {
+    negotiationType = type;
```

Comment:
I wasn't suggesting to add `checkNotNull` elsewhere. All the other places seem they can handle null fine. This one wasn't like that though.